packages/zudoku/src/lib/plugins/openapi/playground/Playground.tsx

```diff
@@ -222,6 +256,10 @@ export const Playground = ({
                 Send
               </Button>
             </div>
+            {/*            <label className="flex gap-2 items-center text-sm">
```

Comment:
Do we need to keep that around?

Reply:
Oops, no. That was just an idea, but don't think we need that right now